janitor/functions.py

```diff
@@ -769,16 +777,15 @@ def filter_on(df, criteria, complement=False):
     .. code-block:: python
 
         df = filter_on(df,
-                       df['value'] < 3,
+                       'value < 3',
                        complement=False)
 
     Method chaining example:
 
     .. code-block:: python
 
         df = (pd.DataFrame(...)
-              .filter_on(df['value'] < 3
-                             complement=False)
+              .filter_on('value < 3, complement=False)
```

Comment:
Missing an apostrophe after `value < 3`.

Reply:
Thanks for the catch, @szuckerman!